testsuite/shell.go

```diff
@@ -128,103 +85,57 @@ func (shell *ShellInstance) ConfigureTypeOfShell(shellName string) {
 		switch runtime.GOOS {
 		case "darwin", "linux":
 			shell.name = "bash"
-			shell.checkExitCodeCmd = fmt.Sprintf(bashExitCodeCheck, exitCodeIdentifier)
 		case "windows":
 			shell.name = "powershell"
 			shell.startArgument = []string{"-Command", "-"}
-			shell.checkExitCodeCmd = fmt.Sprintf(powershellExitCodeCheck, exitCodeIdentifier)
 		}
 	}
-
 	return
 }
 
-func StartHostShellInstance(shellName string) error {
-	return shell.Start(shellName)
+func SetUpHostShellInstance(shellName string) error {
+	return shell.SetUp(shellName)
 }
 
-func (shell *ShellInstance) Start(shellName string) error {
-	var err error
-
+func (shell *ShellInstance) SetUp(shellName string) error {
 	if shell.name == "" {
 		shell.ConfigureTypeOfShell(shellName)
 	}
-	shell.exitCodeChannel = make(chan string)
-
-	shell.instance = exec.Command(shell.name, shell.startArgument...)
-
-	shell.outPipe, err = shell.instance.StdoutPipe()
-	if err != nil {
-		return err
-	}
-
-	shell.errPipe, err = shell.instance.StderrPipe()
-	if err != nil {
-		return err
-	}
-
-	shell.inPipe, err = shell.instance.StdinPipe()
-	if err != nil {
-		return err
-	}
-
-	shell.outScanner = bufio.NewScanner(shell.outPipe)
-	shell.errScanner = bufio.NewScanner(shell.errPipe)
-
-	go shell.ScanPipe(shell.outScanner, &shell.outbuf, "stdout")
-	go shell.ScanPipe(shell.errScanner, &shell.errbuf, "stderr")
-
-	err = shell.instance.Start()
-	if err != nil {
-		return err
-	}
 
 	fmt.Printf("The %v instance has been started and will be used for testing.\n", shell.name)
```

Comment:
This message can probably be dropped. It at least needs to be reworded, as nothing is started anymore.